<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeChatMainWindow</class>
 <widget class="QMainWindow" name="WeChatMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>华文隶书</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>华文隶书</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAddress">
            <property name="font">
             <font>
              <family>华文楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="font">
             <font>
              <family>华文隶书</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEditName">
            <property name="font">
             <font>
              <family>华文隶书</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>华文隶书</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>华文隶书</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLogin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>华文隶书</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>login/register</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="clientState">
        <property name="font">
         <font>
          <family>华文隶书</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>在线状态：离线</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="chatRecord">
        <property name="font">
         <font>
          <family>SimSun-ExtB</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditGroupSend"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGroupSend">
          <property name="font">
           <font>
            <family>华文隶书</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
